apps/api/run-docker.sh

```diff
@@ -0,0 +1,2 @@
+docker compose up
```

Comment:
Issue: I think we still need the `--build` option to rebuild the images when files change.